src/Sarif/NotYetAutogenerated/SarifLog.cs

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) Microsoft.  All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.CodeDom.Compiler;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    /// <summary>
+    /// Static Analysis Results Format (SARIF) Version 1.0.0 JSON Schema: a standard format for the output of static analysis and other tools.
+    /// </summary>
+    [DataContract]
+    [GeneratedCode("Microsoft.Json.Schema.ToDotNet", "0.19.0.0")]
+    public partial class SarifLog : ISarifNode, IEquatable<SarifLog>
+    {
+        /// <summary>
+        /// Gets a value indicating the type of object implementing <see cref="ISarifNode" />.
+        /// </summary>
+        public SarifNodeKind SarifNodeKind
+        {
+            get
+            {
+                return SarifNodeKind.SarifLog;
+            }
+        }
+
+        /// <summary>
+        /// The URI of the JSON schema corresponding to <see cref="Version"/>.
+        /// </summary>
+        [DataMember(Name="$schema")]
+        public Uri SchemaUri { get; set; }
```

Comment:
This file is in NotYetAutogenerated because we don't have a code gen hint that specifies the data member name (the generator always just camel-cases the C# property name), and the property can't be named "$schema".